doc/templates/libdnf5-plugin/template.cpp

```diff
@@ -0,0 +1,91 @@
+#include <libdnf5/base/base.hpp>
+#include <libdnf5/common/exception.hpp>
+
+#include <algorithm>
+
+using namespace libdnf5;
+
+namespace {
+
+/* Enter the name of your new plugin. */
+constexpr const char * PLUGIN_NAME{"template"};
+
+/* Fill in the version of your plugin. */
+constexpr plugin::Version PLUGIN_VERSION{.major = 1, .minor = 0, .micro = 0};
+
+/* Add information about yourself and a description of the plugin. */
+constexpr const char * attrs[]{"author.name", "author.email", "description", nullptr};
+constexpr const char * attrs_value[]{"Fatima Freedom", "dummy@email.com", "Plugin description."};
+
+
+class TemplatePlugin : public plugin::IPlugin {
```

Comment:
I would suggest to add a comment for the class and each of its methods, including inheritance.